.travis.yml

```diff
@@ -11,6 +11,7 @@ before_install:
     - go get -u gopkg.in/alecthomas/gometalinter.v1
     - gometalinter.v1 --install
     - go get -u github.com/go-ini/ini
+    - go get -u github.com/matthewrsj/mixer-tools/swupd
```

Comment:
What error were you getting to need this?

It seems wrong to clone the master repo again when testing a specific PR. If I understand it correctly the sources we need are already in $GOPATH/github.com/clearlinux/mixer-tools/.

Reply:
`swupd` is not currently in clearlinux/mixer-tools so the fullfiles test program failed to compile.

Reply:
The way we are running Travis, it is getting the content of the PR (which contains swupd directory) and puts it inside a directory $GOPATH/src/github.com/clearlinux/mixer-tools, as specified by go_import_path https://docs.travis-ci.com/user/languages/go/#Go-Import-Path.

Reply:
About the tests being executed: the only change we are missing is adding go test call to travis script (as you did in the other patch), but the tests should run just fine: e.g. the PR #15 added the call to go test in travis, and resulted in

    https://travis-ci.org/matthewrsj/mixer-tools/builds/321963818?utm_source=github_status&utm_medium=notification

note how the tests are correctly executed.

Reply:
Yep, you are right. Didn't catch the go_import_path thing. Thanks for explaining.